server/api/controller/cron.js

```diff
@@ -145,6 +146,86 @@ const sendPlanningAlerts = () => {
 		});
 };
 
+const planToEmail = (plan, map) => {
+	return {
+		id: plan.get('id'),
+		title: plan.get('plan_display_name'),
+		city: plan.get('PLAN_COUNTY_NAME'),
+		text: plan.get('goals_from_mavat'),
+		status: plan.get('status'),
+		areaChange: plan.describeHousingChange(),
+		map: 'data:image/gif;base64,'+ map,
+		link:`${this.baseUrl}plan/${plan.get('id')}`
+	};
+}; 
+
+const alertToEmail = (alert) => {
+	const alertTitle = `תוכניות חדשות בקרבת ${alert.get('address') ||  'תחומי הענין שלך'}`;
+	return {
+		alert:{
+			title: alertTitle,
+			unsubscribeLink: `${this.baseUrl}alerts/unsubscribe/${alert.unsubsribeToken()}`
+		}
+	};
+};
+
+const sendDigestPlanningAlerts = async () => {
+	// Send emails for each user, by new plans in his area, that
+	// have been added since he last received a digest email
+	// sendPlanningAlerts(req, res, next) {id
+	Log.info('Running digest send planning alert');
+	const lastSentDifference = 150;
+	const maxAlertsToSend = 5;
```

Comment:
This is not used. Also, why would we send 5 alerts? Shouldn't we send only 1 email?